client/scripts/DrawTogether.js

```diff
@@ -785,6 +789,7 @@ DrawTogether.prototype.setRoom = function setRoom (room) {
 	location.hash = room + "," +
 	                this.paint.public.leftTopX.toFixed() + "," +
 	                this.paint.public.leftTopY.toFixed();
+	this.getFavorites();
```

Comment:
This should be moved to where setRoom is called.
Because the purpose of the setRoom function is to ensure everything that tells the user in what room it is is updated.
So only for the input box, the hash or future things that display the current room.